static/js/nametag.js

```diff
@@ -84,7 +87,7 @@ var nametag = {
         var c_height = this.canvas.height;
         var c_width = this.canvas.width;
         
-        var text = this.name_elem.value;
+        var text = this.name_elem.value + '(' + this.pronoun_elem.value + ')';
```

Comment:
I feel like if we put the PGP on the same line as the name, it will make the already cramped space on the name tag feel even smaller.

Can we maybe move this above or below the name?

Reply:
I've added line breaks around the PGP part of the form. If you view the index.html file you'll see the spacing. It looks clean enough to me. Does it address your concern?